packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -323,6 +323,10 @@ class _PagePosition extends ScrollPositionWithSingleContext implements PageMetri
   // for use when resizing the viewport to non-zero next time.
   double? _cachedPage;
 
+  /// When page view goes ballistic we want to allow interaction with the page
+  @override
+  bool? get preferredBallisticIgnorePointer => false;
```

Comment:
Why only PageView?

Reply:
The property `preferredBallisticIgnorePointer` is still a proof of concept, right now I am trying to make all tests pass and achieve the behavior I want. Then I'll add tests and refactor it. Here the `false` essentially mimics trackpad scrolling as that evaluates to `false` all the time. So for now I just slapped in a `false` as the preference which makes it behave like a trackpad. But with touch gestures.